### DISCUSSION TITLE
how to remove captcha

### DISCUSSION THREAD

**Discussion by prettylittlelies** [Flofy-Dev/rainway-source#24] [Q&A] [answered]
is there any way how to remove captcha? im dumb

**Comment by Flofy-Dev** [comment, marked as answer]
Go to resources/views/auth/register.blade.php and delete Captcha class in the HTML (located at the end of the file/form).
After that, go to app/Http/Controllers/Auth/RegisteredUserController.php, search for the "store" function and in "$request->validate" (without the quotation marks), and delete the "g-recaptcha-response" line (it's line 40 in the file). Now you can use the register function without the captcha, sorry for the late response!
